### PR TITLE
Updated fr (French) translation

### PR DESCRIPTION
**Pull request by StelFux**
I'm a beginner in the use of git! 
This is a version containing only the strings.xml file

### CODE REVIEW

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -206,6 +215,9 @@
     <string name="pref_name_ui_disableTouch_keyCode">Disable touch input button</string>
     <string name="pref_desc_ui_disableTouch_keyCode">Press here to configure the button used to disable/enable touch input</string>
 
+    <string name="pref_name_ui_scrollOverBottom">Faire défiler vers le bas pour marquer comme lu</string>
+    <string name="pref_desc_ui_scrollOverBottom">Appuyez ici pour configurer le nombre de fois que vous avez besoin de \"faire défiler vers le bas\" pour marquer un article comme lu</string>
```
[the remark this answers is not in the document]
Comment (StelFux) [reply app/src/main/res/values-fr/strings.xml]: Thank you !

File: app/src/main/res/values-fr/strings.xml [hunk 2]
```diff
@@ -192,6 +197,10 @@
     <string name="pref_desc_ui_article_fontSerif">Police avec empattement (Serif)</string>
     <string name="pref_name_ui_article_textAlignment_justify">Alignement du texte : justifié</string>
     <string name="pref_desc_ui_article_textAlignment_justify">Aligne le texte pour que chaque ligne ait la même longueur (comme dans les journaux ou les magazines)</string>
+
+    <string name="pref_name_ui_article_fullscreen">Voir l\'article en écran</string>
```
[the remark this answers is not in the document]
Comment (StelFux) [reply app/src/main/res/values-fr/strings.xml]: Thank you !

**Comment by Kdecherf** [comment]
Hello @StelFux,

Thanks for your update and welcome on git and github :)

I let another people approve the change ;-)

**Comment by Kdecherf** [comment]
@Strubbl no doubt, you can merge :)

**Comment by Strubbl** [comment]
@StelFux thanks a lot for your contribution

**Pull request merged**